amethyst_ui/src/font/default.rs

```diff
@@ -0,0 +1,26 @@
+use amethyst_assets::{AssetStorage, Loader, SimpleFormat};
+use format::{FontAsset, FontHandle, TtfFormat};
+use font::systemfont::default_system_font;
+use font_kit::handle::Handle as FontKitHandle;
+
+/// Get the system default fonts.
+/// If unable to, gets the local square.ttf font.
+pub fn get_default_font(loader: &Loader, storage: &AssetStorage<FontAsset>) -> FontHandle {
+	let system_font = default_system_font();
+
+	match system_font {
+		Ok(handle) => match handle {
+			FontKitHandle::Path{..} => unimplemented!(),
```

Comment:
It would probably be good to include a message when using `unimplemented` so that anyone who hits this case get's a more useful message, e.g. `unimplemented!("Default system font was provided as a path, this is not yet supported")`.